app/controllers/admin/web_hooks_controller.rb

```diff
@@ -0,0 +1,46 @@
+class Admin::WebHooksController < Admin::AdminController
+
+  def index
+    data = {
+      web_hook_event_types: WebHookEventType.all,
+      web_hooks: WebHook.all # TODO: Pagination?
+    }
+    render_serialized(OpenStruct.new(data), AdminWebHooksSerializer, rest_serializer: true)
```

Comment:
I was thinking I could sideload all relevant event types like `Badge` models. Yes, I would go for a static list for Ember now.